src/typing_extensions.py

```diff
@@ -984,13 +1042,27 @@ def __new__(cls, name, bases, ns, *, total=True, closed=False):
                 optional_keys.update(base_dict.get('__optional_keys__', ()))
                 readonly_keys.update(base_dict.get('__readonly_keys__', ()))
                 mutable_keys.update(base_dict.get('__mutable_keys__', ()))
-                base_extra_items_type = base_dict.get('__extra_items__', None)
-                if base_extra_items_type is not None:
-                    extra_items_type = base_extra_items_type
+                if getattr(base, "__closed__", None) and not closed:
```

Comment:
```suggestion
                if getattr(base, "__closed__", None) and closed is not None:
```

We want to not pass closed.

Reply:
Thanks, going to remove this check instead as you hinted at. I wanted to do this in https://github.com/python/peps/pull/4166/files but forgot to update this PR.